compiler/circle-mpqsolver/src/pattern/PatternResolver.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __MPQSOLVER_PATTERN_RESOLVER_H__
+#define __MPQSOLVER_PATTERN_RESOLVER_H__
+
+#include <core/Quantizer.h>
+
+#include <luci/IR/Module.h>
+
+namespace mpqsolver
+{
+namespace pattern
+{
+
+using LayerParam = luci::CircleQuantizer::Options::LayerParam;
+
```

Comment:
Please do not use `using` in the header file.

Reply:
ok